src/display.rs

```diff
@@ -282,8 +282,14 @@ impl Display {
             new_size = Some(sz);
         }
 
+        // Font size modification detected
         if terminal.font_size_modifier != self.font_size_modifier {
-            // Font size modification detected
+            // Make sure the font size is at least 1
+            let config_font_size = config.font().size().as_f32_pts() as i8;
```

Comment:
The font size doesn't decrease beyond minimal value anymore, but if I hold <kbd>Ctrl +</kbd>, I can overflow this value (because the largest font size is `274`) and thus at some point the font size resets to the minimal value and starts growing from scratch 😄

Reply:
Thanks, should be fixed now.

Reply:
Yup, perfect now 👍